Formula/geocode-glib.rb

```diff
@@ -17,22 +18,21 @@ class GeocodeGlib < Formula
   depends_on "gobject-introspection" => :build
   depends_on "meson" => :build
   depends_on "ninja" => :build
-  depends_on "pkg-config" => :build
+  depends_on "pkg-config" => [:build, :test]
   depends_on "glib"
   depends_on "gtk+3"
```

Comment:
Might be able to use `gtk4` instead? Not sure exact usage of dependency, but it looks to me that it is for updating icon cache in postinstall. Given no `:postinstall`-only dependencies, we end up needing an unnecessary runtime dependency.

At least there doesn't seem to be a GTK dependency in Arch's and Debian's package.